portfolio/src/main/webapp/script.js

```diff
@@ -253,21 +253,24 @@ function drawMinChart() {
   drawChart(false);
 }
 
-var markers = [];
-var map;
-
+/*
+ * Load Google Maps at a world zoom level and adds markers to all 
+ * latitude/longitude locations.
```

Comment:
This comment doesn't indicate that the markers are added in a particular way (one by one, with a sleep in between).

Reply:
added more description!